stories/Templates/Authors/Authors.jsx

```diff
@@ -1,56 +1,70 @@
-import React, { useEffect } from 'react';
-import logowhite from '../../assets/images/undp-logo-white.svg';
-import logo from '../../assets/images/undp-logo-blue.svg';
-import user from '../../assets/images/user.svg';
-import viewport from '../../assets/js/viewport';
-import { Heading } from '../../Atom/Typography/Heading/Heading';
-import CustomSelect from '../../Components/Forms/Dropdowns/CustomSelect/CustomSelect';
-import { Breadcrumbcomponent } from '../../Components/Navigationcomponents/Breadcrumbs/Breadcrumbs';
-import GlobalHeader from '../../Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader';
-import { AuthorSummary } from '../../Organism/Blocks/Authorcard/AuthorSummary';
-import { ContentCard } from '../../Organism/Blocks/ContentCard/ContentCard';
-import { Footer } from '../../Organism/Footer/Footer';
-import './Authors.scss';
-import init from '../../assets/js/select-init';
-import { Authorcard } from '../../Components/UIcomponents/Author/Authorcard/Authorcard';
-import { CtaButton } from '../../Components/UIcomponents/Buttons/Cta_button/CtaButton';
-import { SearchExpand } from '../../Components/Forms/expandable search/SearchExpand';
+import React, { useEffect } from "react";
+import logowhite from "../../assets/images/undp-logo-white.svg";
+import logo from "../../assets/images/undp-logo-blue.svg";
+import user from "../../assets/images/user.svg";
+import viewport from "../../assets/js/viewport";
+import { Heading } from "../../Atom/Typography/Heading/Heading";
+import CustomSelect from "../../Components/Forms/Dropdowns/CustomSelect/CustomSelect";
+import { Breadcrumbcomponent } from "../../Components/Navigationcomponents/Breadcrumbs/Breadcrumbs";
+import GlobalHeader from "../../Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader";
+import { AuthorSummary } from "../../Organism/Blocks/Authorcard/AuthorSummary";
+import { ContentCard } from "../../Organism/Blocks/ContentCard/ContentCard";
+import { Footer } from "../../Organism/Footer/Footer";
+import "./Authors.scss";
+import init from "../../assets/js/select-init";
+import { Authorcard } from "../../Components/UIcomponents/Author/Authorcard/Authorcard";
+import { CtaButton } from "../../Components/UIcomponents/Buttons/Cta_button/CtaButton";
+import { SearchExpand } from "../../Components/Forms/expandable search/SearchExpand";
 
 const Authors = ({
-  breadcrumbData, authorSummeryData, contentCardData, footerData,
-  sortText, viewText, closeText,
-  undpheading, text, authorData, headingData, buttontype, buttonData,  languageswitcherData,
+  breadcrumbData,
+  footerData,
+  sortText,
+  viewText,
+  closeText,
+  undpheading,
```

Comment:
@Ashish-srij I do not see `undpheading` being used anywhere. Can you remove it if it is not required.

Reply:
@shishirphadke I have made all the changes.